mica-core/src/main/java/org/obiba/mica/access/service/DataAccessEntityService.java

```diff
@@ -453,4 +482,47 @@ private boolean isDataAccessAgreementContext(Map<String, String> ctx) {
   private boolean isDataAccessPreliminaryContext(Map<String, String> ctx) {
     return ctx.containsKey("type") && ctx.get("type").equals(DataAccessPreliminary.class.getSimpleName());
   }
+
+  /**
+   * @return principals with READER permission (VIEW action) excluding applicant, collaborators, MICA_ADMIN and MICA_DAO
+   */
+  private Map<String, List<String>> getRequestReaders(Map<String, String> ctx, String[] applicantsAndCollaborators) {
+    Map<String, List<String>> map = new HashMap<>();
+    map.put("users", new ArrayList<>());
+    map.put("groups", new ArrayList<>());
+
+    List<String> excludedPrincipals = new ArrayList<>();
+    excludedPrincipals.addAll(Arrays.asList(applicantsAndCollaborators));
```

Comment:
It would be safer to register both the principal and its type like: 

`subject_type:subject_principal` 

(example: USER:michel or GROUP:mica-administrator)

Reply:
Old Mica versions that had the DAR list under admistration could. I had an acl where the instance was specific (was an id instead of `*`) and wanted to make sure those were included.

But thinking more about it, since it can't be accessed anymore, I will remove that.

Else currently we do still have 
![image](https://user-images.githubusercontent.com/6180385/213656195-f8bc598f-e46f-46d9-925c-97de1001fa5a.png)
 which gives permission to all DAR, their private comments and action logs.

Reply:
It would be interesting to have a way of setting that kind of permissions from the DAR's (template) page.

Reply:
For the permissions on the `*` instance?
The collaborator feature does give editor permissions to a specific DAR already.

Reply:
I was thinking of a kind of "data access review assignment". But this assignment could grant more possible actions than just reading a DAR. Let's think about that later.

Reply:
Yes, some partners do want to have a "committee" aside from the DAO that helps in the review process of a DAR.

Reply:
That is what I expected:

```
excludedPrincipals.addAll(Arrays.asList(applicantsAndCollaborators).stream().map(principal -> SubjectAcl.Type.USER + ":" + principal).collect(Collectors.toList()));
excludedPrincipals.add(SubjectAcl.Type.GROUP + ":" + Roles.MICA_ADMIN);
excludedPrincipals.add(SubjectAcl.Type.GROUP + ":" + Roles.MICA_DAO);
```

and then you can safely exclude acls based on the type and the principal:

```
specificFoundAcls.stream().filter(acl -> acl.hasAction("VIEW")).filter(acl -> !excludedPrincipals.contains(acl.getType() + ":" + acl.getPrincipal()))
etc...
```